src/inverseRobot.js

```diff
@@ -7,7 +7,15 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
-}
+  const inverse = {};
```

Comment:
Its better to make more proper name for new object. 
everithing else is perfect.

Reply:
As for me, not so bad name for variable) But `inverseRobot` looks better